helpers.rb

```diff
@@ -0,0 +1,35 @@
+module Helpers
+  def response(name)
+    sleep 1
+    puts "\n#{name} created successfully!!!"
+    sleep 1
+    continue?
+  end
+
+  def continue?
+    print "\nDo you wish to continue? [Y/N]: "
+    answer = gets.chomp
+
+    if answer.downcase == 'y' || answer.downcase == 'yes' || answer == ''
+      clear
+      prompt_user
+    else
+      exit
+    end
+  end
+
+  def invalid_prompt
+    clear
+    puts 'Incorrect selection, please try again!'
+    sleep 1
+  end
+
+  def clear
+    print "\e[2J\e[f"
+  end
+
+  def exit
+    clear
+    nil
+  end
```

Comment:
Overall, you have made a great effort with the UI structure, file organization, dividing your methods into to modules, clean code, and best practices 👏🏾 
So far the only thing that is working on your UI is the exit option. Please make sure that everything is working properly and behaves in the same way as the [example video](https://www.youtube.com/watch?v=vkkgrhD6aXQ).

Reply:
Oh nevermind. I just remembered that we created methods in the previous project that should allow us to implement this. Thank you!